starling/display/Image.hx

```diff
@@ -274,6 +276,7 @@ class Image extends Quad
         vertexData[6] = point.x;
         vertexData[7] = point.y;
         
+        @:privateAccess mBatcherQuadTextureData.data = mTexture.base.__getTexture();
```

Comment:
As you said, this call might be cheap, but I agree, it feels a bit wrong here. Because I don't have a great idea in mind, I will think about it until tomorrow.

Adding a context created listener here to set `mVertexDataCacheInvalid = true` also feels like quite some change. Not sure...

Reply:
One of the possible options would be to introduce a "texture uploaded" callback from the Texture and handle that from within Image, updating/recreating the `QuadTextureData`, but also not sure...